### ISSUE TITLE
SPIKE Hearing Worksheet: Display an Issue's Rating Percentage from VBMS

### ISSUE THREAD

**Issue by lakohl** [department-of-veterans-affairs/caseflow#3785]
Judges have indicated that they would like to see the rating percentage for issues that is displayed in the VBMS sidebar.

Investigate where this info is stored and how we might be able to pull it in.

**Comment by lakohl** [comment]
flagging @annienguyenUSDS for awareness

**Comment by aroltsch** [comment]
@nicholasholtz said that it would be super useful for a judge to see all these issues with a percentage and an effective date. Waiting on VBMS dev team to respond.

**Comment by aroltsch** [comment]
VBMS team responded:
```Yes. You can use the getRating operation in the BGS RatingManagementService

http://beplinktest.vba.va.gov:80/RatingManagementService/RatingManagementService

The percentage for each Rating Issue is returned in the response.```

**Comment by aroltsch** [comment]
We should be able to use readCurrentRatingProfileByPtcpntID in RatingManagementService
http://beplinktest.vba.va.gov/RatingInformationService/RatingInformationService?wsdl

Open Questions:
1. Is participant ID always veteran's  BFCORLID or can it be appellant's ID? How do we get appellant's BFCORLID?
2. In VBMS, what we see is called disabilities. Can we map Vacols issues to BGS/VBMS disabilities?
3. Can we display BGS issues/disabilities in a different section on the worksheet page instead of the issues section so a judge can reference them if needed?

**Comment by aroltsch** [comment]
http://beplinktest.vba.va.gov/RatingInformationService/RatingInformationService?wsdl
If order to use this service in production, we need to ask Nikita to contact BGS team to open  up the service for us. Here is the PR: https://github.com/department-of-veterans-affairs/ruby-bgs/pull/42

**Comment by lakohl** [comment]
@sharon would you mind following up on this?

**Comment by lakohl** [comment]
There are two parts to this: 1)an overall Service Connection Rating for a Veteran (shows in the VBMS sidebar) and 2)a table where a disability has a diagnostic code that can be matched to the diagnostic code on an appeal stream issue's Levels 1-3.

**Comment by annienguyenUSDS** [comment]
@tejans24, could you please post the latest info you have on this? Did you get through BGS documentation or got stuck?

**Comment by tejans24** [comment]
I got stuck, I emailed the vbms team regarding the issue but I didn't receive a reply. I will create a issue today on https://github.com/department-of-veterans-affairs/dsva-vbms to see if there is better luck.

**Comment by annienguyenUSDS** [comment]
@tejans24, can you link to the ticket number for this?

**Comment by annienguyenUSDS** [comment]
@tejans24, can't find the ticket for this if you can create and link one. thanks!

**Comment by tejans24** [comment]
@annienguyenUSDS A new issue is in the https://github.com/department-of-veterans-affairs/dsva-vbms repository #101

**Comment by tejans24** [comment]
I talked to Shane about this. It looks like the intake team is working on getting BGS to open a few end points for ratings. He mentioned that this might take from 1-3 months.

**Comment by annienguyenUSDS** [comment]
Shane is looking into the BGS API, and Teja will get access to VBMS.
